docs/pages/foundations/iconography/library.js

```diff
@@ -53,72 +54,79 @@ type IconData = {|
     | 'Time'
     | 'Toggle'
     | 'Utility and tools',
+  description: string,
 |};
 
-function IconTile({ iconName, onTap }: {| iconName: IconName, onTap: () => void |}) {
+function IconTile({
+  iconName,
+  iconDescription = 'Description coming soon',
```

Comment:
Another possibility: conditionally wrap the content with Tooltip based on the presence of iconDescription (might be less noisy in the short term)

Reply:
I thought about that, but given description is required, and Layla's making the updates right after, I think it'll be ok for a day